src/components/FloatingHelper/content/HelperContent.st.css

```diff
@@ -0,0 +1,12 @@
+:import {
+  -st-from: "../../../palette.st.css";
+  -st-named: brandSecondary, backgroundSecondary
+}
+
+:import {
+  -st-from: "../../../typography.st.css";
```

Comment:
Unused imports. this whole file is not needed

Reply:
removed all imports